### DISCUSSION TITLE
Show message body in notification drawer

### DISCUSSION THREAD

**Discussion by AxeemHaider** [directus/directus#19138] [Feature Requests]
### Summary

I think it's good to show notification message body in notification drawer. Currently it only show the notification subject and time. 
There should be **Read more** or something like that when user click on item, open notification body. And then mark it as read automatically. 

### Motivation

I think all other notification design system work like that.

### Detailed Design

Currently it only show the notification subject and time. 
There should be **Read more** or something like that when user click on item, open notification body. And then mark it as read automatically. 

### Must Have:
- Notification Message body

### Drawbacks

I don't thing there is any drawback for this.

### Alternatives

Alternatives we can send email with notification as we already doing this.

### Adoption Strategy

No other effect, it can be easily adop

**Comment by rijkvanzanten** [comment]
This was implemented in https://github.com/directus/directus/pull/19503 🥳
